docs/reference/docs/index_.asciidoc

```diff
@@ -155,12 +155,8 @@ than the version of the stored document. If there is no existing document
 the operation will succeed as well. The given version will be used as the new version
 and will be stored with the new document. The supplied version must be a non-negative long number.
 
-`force`:: the document will be indexed regardless of the version of the stored document or if there
-is no existing document. The given version will be used as the new version and will be stored
-with the new document. This version type is typically used for correcting errors.
-
-*NOTE*: The `external_gte` & `force` version types are meant for special use cases and should be used
-with care. If used incorrectly, they can result in loss of data.
+*NOTE*: The `external_gte` version type is meant for special use cases and
+should be used with care. If used incorrectly, it can result in loss of data.
```

Comment:
I think `force` should still be in the docs but it should be super clear that it is a horrible choice. If we just drop it from the docs someone is going think it is a hidden feature. I think if we did something like "There is another option `force` which is deprecated because it can result in the primary and replica diverging." That way it is still documented as a bad choice but it isn't in the list of options.